SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -56,17 +56,17 @@ extension Station: VOKMappableModel {
         return { (_, inputObject: NSManagedObject) in
             guard let
```

Comment:
We'll also need to update our coding standards for this, but I feel like I'm missing something: Xcode isn't syntax coloring the `let` statements the same

![screen shot 2016-09-23 at 9 02 53 am](https://cloud.githubusercontent.com/assets/163838/18788475/9e729952-816c-11e6-8346-001d62cd9d01.png)